docs/user/container.md

```diff
@@ -156,7 +156,7 @@ Layout Type           unknown (0)
 Layout Version        1
 Max Snapshot          0
 Owner                 jlombard@
-Redundancy Factor     rf0
+Redundancy Factor     rd_fac0
```

Comment:
is the value rd_fac0 or just 0 (same question for other values below)?

Reply:
The ouput of the command is valid:
![image](https://user-images.githubusercontent.com/94527853/214609757-c7e94057-d963-42da-8b10-68b324d88e66.png)

Reply:
ok thanks for checking. it might make sense to not include the property name in the value itself. but this is unrelated to this PR.

Reply:
Yep, it will need some works... expecially for updating all the tests as the json output is also printing the same value.